tools/floodgate/js/config.js

```diff
@@ -0,0 +1,32 @@
+import {
+  getUrlInfo,
+} from '../../loc/utils.js';
+import {
+  fetchConfigJson,
+  getSharepointConfig,
+  getHelixAdminConfig,
+} from '../../loc/config.js';
+
+const FLOODGATE_CONFIG = '/drafts/floodgate/configs/config.json';
```

Comment:
@amauch-adobe It is a temp location for now similar to [Milo Loc](https://github.com/adobecom/milo/blob/main/tools/loc/config.js#L15). It will most probably move to AIO in the near future once we have a better location for storing configs like these.

Reply:
@janaki-r-bhagwath Agree, both FG and loc configs should be moved to a better location.

In case of Floodgate, it is just the "helix-sp" sheet info that is needed with some FG specific stuff which is what I have added in a separate config excel file [here](https://adobe.sharepoint.com/:x:/r/sites/adobecom/_layouts/15/Doc.aspx?sourcedoc=%7B8215365C-10F4-48D7-A41D-FB4B239DF6C6%7D&file=config.xlsx&action=default&mobileredirect=true)

In case of FG, as most of the info in that sheet are just required to run the sharepoint APIs, ideally all of it can be moved to AIO (or perhaps in a central location that AIO can access). This may not apply to loc.

Perhaps we will have a better understanding of where this central location for loc and FG configs can be once we start exploring the AIO side of things.